.github/ISSUE_TEMPLATE.md

```diff
@@ -14,7 +14,7 @@
 
 For documentation-related issues, you can check the latest versions of the docs on `master` here:
 
-https://pandas-docs.github.io/pandas-docs-travis/
+https://pandas.pydata.org/pandas-docs/stable/
```

Comment:
I think this shouldn't be stable, but `dev`. As I mentioned in the other comment, not really sure about the exact path.

Reply:
> I think this shouldn't be stable, but `dev`. As I mentioned in the other comment, not really sure about the exact path.

@datapythonista does https://pandas.pydata.org/docs/ is the path you meant?

Reply:
Probably, looks like this is what we're using. Not sure where I've got the `pandas-docs` from. I guess I've got confused.